sdk/src/rest/commands/delete/comments.ts

```diff
@@ -0,0 +1,38 @@
+import type { DirectusComment } from '../../../schema/comment.js';
+import { throwIfEmpty } from '../../utils/index.js';
+import type { RestCommand } from '../../types.js';
+
+/**
+ * Delete multiple existing comments.
+ * @param keys
+ * @returns
+ * @throws Will throw if keys is empty
+ */
+export const deleteComments =
+	<Schema>(keys: DirectusComment<Schema>['id'][]): RestCommand<void, Schema> =>
```

Comment:
Since the DELETE endpoint also supports updating by Query we should reflect that in 
```suggestion
	<Schema>(keysOrQuery: DirectusComment<Schema>['id'][] | Query<Schema, DirectusComment<Schema>>): RestCommand<void, Schema> =>
```

Reply:
Updated via https://github.com/directus/directus/pull/22295/commits/40fe2953e86ecd9476805a79a6932a2fa47fc6b8